src/frame.js

```diff
@@ -2,13 +2,21 @@
 
 import fs from 'fs';
 import gmBase from 'gm';
-import Promise from 'bluebird';
 import tempWrite from 'temp-write';
 import getPixelsCb from 'get-pixels';
 import DevtoolsTimelineModel from 'devtools-timeline-model';
 
 const gm = gmBase.subClass({imageMagick: true});
-const getPixels = Promise.promisify(getPixelsCb);
+const getPixels = (buff) => {
```

Comment:
You could use [`pify`](https://github.com/sindresorhus/pify) here. It's a tiny package without any dependencies.